sycl/test/devicelib/math_utils.hpp

```diff
@@ -1,25 +1,33 @@
 #ifndef MATH_UTILS
 #include <complex>
 #include <limits>
-using namespace std;
 // T must be float-point type
 template <typename T>
 bool is_about_FP(T x, T y) {
```

Comment:
While you are here, could you add a comment explaining what this function... is_about? :-)

Reply:
Hi, Ronan.
Since it is not proper to compare float point using operator ==, this function measure whether the result of cmath function from kernel is close to the reference and machine epsilon is used as threshold in this function.
Thanks very much.

Reply:
Ah quite clearer with a comment! :-)

Then perhaps a clearer name for the function would help...

`fuzzy_equal_fp` ? `approx_equal_fp` ? I am not an native English speaker so I am not very helpful here...